### ISSUE TITLE
Xposed on Xposed 2

### ISSUE THREAD

**Issue by netstu** [rovo89/XposedBridge#131]
i readed the page https://github.com/rovo89/XposedBridge/issues/41

but i don't know,  

1. right classloader  ? what classloader is right classloader?

2. suitable time ? 

i try to hook the other xposed module, but output errors, not fuond the class

**Comment by rovo89** [comment]
1. The ClassLoader that XposedBridge used to load the module.
2. After the other module has loaded, but before the function you want to hook has been executed.

As mentioned in the other issue, that's pretty hard to do. There's no API to retrieve the ClassLoader, you would have to use reflection APIs to get them and hope that the internal implementation of Xposed doesn't change. There's no way to say that your module should be loaded after another one.

In short: Don't do it. Talk to the developer of the other module. In the worst case, fork it and add your own logic.

**Comment by netstu** [comment]
thanks

i can't fork the source code, that xposed module is not open source
